### DISCUSSION TITLE
Gaomon S620 not detected

### DISCUSSION THREAD

**Discussion by TheJohnThatGae** [OpenTabletDriver/OpenTabletDriver#1191] [Q&A] [answered]
When I downloaded this driver and saw that it supports the Gaomon S620 I immediately downloaded it and booted it up but the tablet wasn't detected in the software any explanation?

**Comment by InfinityGhost** [comment, marked as answer]
https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ#my-gaomonhuion-tablet-isnt-detected-but-is-listed-as-supported

**Reply by jamesbt365** [reply]
This or a missing config folder.
